src/xz/file_io.c

```diff
@@ -39,6 +39,30 @@ static bool warn_fchown;
 
 #include "tuklib_open_stdxxx.h"
 
+#ifdef _MSC_VER
+#define close	_close
+#if _MSC_VER >= 1500
```

Comment:
VS older than 2013 update 2 cannot build anything from XZ Utils so this `#if` isn't needed. In practice maybe even VS2013 isn't worth worrying as it will go out of support in April 2024.